src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/config.json

```diff
@@ -0,0 +1,126 @@
+﻿{
```

Comment:
Why is there a config.json in here? shouldn't that be repo-specific? Or maybe this is just a sample one?